app/views/rides/index.html.slim

```diff
@@ -1,3 +1,4 @@
+= flash[:notice] if !flash[:notice].nil?
```

Comment:
We prefer to use the positive version which would be `if flash[:notice].present?` or `if flash[:notice]` or you could use `unless` instead of `if !`